cloudbuild.yaml

```diff
@@ -36,6 +36,14 @@ steps:
     - GIT_TAG=$_GIT_TAG
     - EXTRA_TAG=$_PULL_BASE_REF
     - DOCKER_BUILDX_CMD=/buildx-entrypoint
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20240718-5ef92b5c36
+    entrypoint: make
+    args:
+      - bbr-helm-chart-push
```

Comment:
you can add this to the args above for the inferencepool helm push, so basically the make will run for both, no need for a separate step.

I think we can consolidate the bbr and epp images too, but we can do that separately

Reply:
So just to confirm, `make inferencepool-helm-chart-push` will push charts for both inferencepool and bbr?

Reply:
Oh nvmd, I think I see what you mean.

Reply:
Done